templates/solutions/infrastructure/_form_infrastructure.html

```diff
@@ -0,0 +1,89 @@
+<div class="p-modal" style="display: none;" id="infrastructure-modal">
+  <div class="p-modal__dialog is-paper" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+      <div class="row--50-50 p-strip is-shallow">
+        <div class="col">
+          <h2 class="p-heading--1">Contact Canonical</h2>
+          <p>Fill in this form and a member of our team will be in touch shortly.</p>
+        </div>
+        <div class="col">
+          <button class="p-modal__close p-modal-close-button u-no-margin--top" aria-controls="infrastructure-modal" aria-label="Close active modal">Close</button>
+          <form action="https://ubuntu.com/marketo/submit" method="post" class="modal-form">
+            <ul class="p-list">
+              <li class="p-list__item">
+                <label for="firstName" class="is-required">First name:</label>
+                <input required id="firstName" name="firstName" maxlength="255" type="text"/>
+              </li>
+              <li class="p-list__item">
+                <label for="lastName" class="is-required">Last name:</label>
+                <input required id="lastName" name="lastName" maxlength="255" type="text"/>
+              </li>
+              <li class="p-list__item">
+                <label for="email" class="is-required">Email:</label>
+                <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              {% with required="true" %}
+                {% include "shared/forms/_country.html" %}
+              {% endwith %}
```

Comment:
mm for some reason country is not showing up as a dropdown, but a text field?

Reply:
Happy to file a maintenance ticket, but again this is not within scope for this pr see [comment](https://github.com/canonical/canonical.com/pull/1222/files#r1531772021).